.vscode/settings.json

```diff
@@ -0,0 +1,10 @@
+{
+    "restructuredtext.confPath": "${workspaceFolder}\\docs\\source",
+    "python.formatting.blackArgs": [
+        "--line-length",
+        "80",
+        "--skip-string-normalization"
+    ],
+    "python.formatting.provider": "black",
+    "editor.rulers": [80, 88]
+}
```

Comment:
That's ok (as you did ask) - I have used VSCode occasionally, I just prefer PyCharm.

Reply:
Cool, just wanted to make sure it's okay being there =)